<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCL_Labeller</class>
 <widget class="QMainWindow" name="PCL_Labeller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>817</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_12" stretch="9,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QVTKWidget" name="qvtkWidget" native="true"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QTabWidget" name="list_tabWidget">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="file_tab">
             <attribute name="title">
              <string>File List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QListWidget" name="file_listWidget">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="label_tab">
             <attribute name="title">
              <string>Label List</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QListWidget" name="label_listWidget">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                  <italic>false</italic>
                 </font>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Copy from Prev</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Reserve</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10" stretch="2,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,1">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Label</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="label_le">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Center - X</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="centerx_dsb">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="suffix">
                   <string> m</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>-999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Center - Y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="centery_dsb">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="suffix">
                   <string> m</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>-999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Center - Z</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="centerz_dsb">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="suffix">
                   <string> m</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>-999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Width - X</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="widthx_dsb">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="suffix">
                   <string> m</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,0">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Height - Y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="heighty_dsb">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="suffix">
                   <string> m</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,0">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>Depth - Z</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="depthz_dsb">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="suffix">
                   <string> m</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,2,1,2,1,2,1,2">
              <item>
               <widget class="QPushButton" name="prevPCD_pb">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="save_pb">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="newLabel_pb">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>New Label</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="deleteLabel_pb">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Delete Label</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="nextPCD_pb">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&gt;&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
